app/views/user_mailer/new_comment.html.erb

```diff
@@ -4,7 +4,7 @@
   plan_title = @plan.title
   user_name = @plan.owner.name
   helpdesk_email = Rails.configuration.branding[:organisation][:helpdesk_email]
-  contact_us_url = Rails.configuration.branding[:organisation][:contact_us_url]
+  contact_us = Rails.configuration.branding[:organisation][:contact_us_url] || contact_us_url
```

Comment:
that change should be sufficient for avoiding the override of contact_us_url helper. The one defined at the .yml it is within a hash so no problem should arise.